src/genie/libs/parser/ironware/tests/ShowIPRoute/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,188 @@
+expected_output = {
```

Comment:
this is `default` vrf routing table? like other comment, would suggest to have `vrf` top level.

Reply:
@tahigash Yes, similar to above however the command is `show ip route vrf xx`, it can be assumed when running `show ip route` that the user is looking at the default table.

Reply:
ok. if so, can you add `vrf` level like ospf parser? it's same reason with the ospf parser.